src/components/button-group/button-group.ts

```diff
@@ -56,6 +56,7 @@ export default class SlButtonGroup extends LitElement {
   }
 
   render() {
+    // eslint-disable-next-line lit-a11y/mouse-events-have-key-events -- focusout & focusin support bubbling where as focus & blur do not which is necessary here
```

Comment:
Just nitpicking here 😛 

```suggestion
    // eslint-disable-next-line lit-a11y/mouse-events-have-key-events -- focusout & focusin support bubbling whereas focus & blur do not which is necessary here
```